docker/usr/local/etc/php/php.ini

```diff
@@ -59,7 +59,7 @@ request_order = "GP"
 register_argc_argv = Off
 auto_globals_jit = On
 
-post_max_size = ${PHP_INI_POSTMAX_SIZE}
+post_max_size = ${PHP_INI_POST_MAX_SIZE}
```

Comment:
good catch! Please add the possibility to make use of both variants. Else this would be a breaking change and as we follow semver, we would be required to release a new major version.
In addition, there is a `TODO.md` and at the bottom we collect all TODOs for the next major version. Please add there, that we have to remove the fault variable name.

And please fill out the CHANGELOG and add patch bump the version

Reply:
This is the only place where the variable needs to be changed.
I think a normal Merge is enough.

Reply:
a merge would break all child images, which use `PHP_INI_POSTMAX_SIZE` instead of the new bugfixed version